grakn-engine/src/main/java/ai/grakn/engine/lock/JedisLock.java

```diff
@@ -45,11 +46,12 @@
     private Pool<Jedis> jedis;
     private Lock lock = new ReentrantLock();
 
-    public JedisLock(Pool<Jedis> jedis, String lockName) {
+    public JedisLock(Pool<Jedis> jedis, String internalLockName) {
         Preconditions.checkNotNull(jedis,"JedisPool used in lock cannot be null");
-        Preconditions.checkArgument(lockName != null && !lockName.isEmpty(),"Lock name not valid");
+        Preconditions.checkArgument(internalLockName != null && !internalLockName.isEmpty(),"Lock name not valid");
         this.jedis = jedis;
-        this.lockName = lockName;
+        this.lockName = internalLockName;
```

Comment:
The get takes the unprefixed version, it breaks things if we return the prefixed version